Jellyfin.Plugin.Reports/Api/Data/ReportExport.cs

```diff
@@ -1,37 +1,42 @@
 ﻿#nullable disable
 
-using System.Globalization;
+using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using Jellyfin.Plugin.Reports.Api.Model;
 
 namespace Jellyfin.Plugin.Reports.Api.Data
 {
     /// <summary> A report export. </summary>
-    public class ReportExport
+    public static class ReportExport
 	{
 		/// <summary> Export to CSV. </summary>
 		/// <param name="reportResult"> The report result. </param>
 		/// <returns> A string. </returns>
-		public string ExportToCsv(ReportResult reportResult)
+		public static string ExportToCsv(ReportResult reportResult)
 		{
-			StringBuilder returnValue = new StringBuilder();
+			static void AppendRows(StringBuilder builder, List<ReportRow> rows)
+			{
+				foreach (ReportRow row in rows)
+				{
+					builder.AppendJoin(';', row.Columns.Select(s => s.Name.Replace(',', ' '))).AppendLine();
```

Comment:
Possible to change this to another separator like a Pipe | instead of semicolon? ; does introduce issues with fields that contrain semicolon in the data fields.